src/k32w1/k32w1/K32W1480_connectivity.ld

```diff
@@ -146,15 +146,15 @@ TARGET(binary)
  * "NBU_BINARY_IMAGE_PATH" is defined for specifying path to connectivity_test.bin binary ... is set to SDK path at this moment:
  * $ENV{NXP_K32W1_SDK_ROOT}/middleware/wireless/ieee-802.15.4/boards/k32w1/connectivity_test_cm3/bm/iar/Debug/Exe/
  */
-INPUT(k32w1_nbu_ble_15_4_dyn_1_9_10_0.sb3)
+INPUT(k32w1_nbu_ble_15_4_dyn_1_9_11_1.sb3)
```

Comment:
You should remove it since you need to load the .sb3 file with blhost as is written in the readme